proto/cmp/types/v1alpha/bookable.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package cmp.types.v1alpha;
+
+import "cmp/types/v1alpha/time.proto";
+
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/bookable.proto.dot.xs.svg)
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/types/v1alpha/bookable.proto.dot.svg)
+message Bookability {
+  Bookable bookable = 1;
+
+  // In case of a bookable=on_request status, a confirmation time should be
+  // given
+  cmp.types.v1alpha.Time confirmation_time = 2;
+}
+
+enum Bookable {
+  BOOKABLE_UNSPECIFIED = 0;
+  BOOKABLE_AVAILABLE = 1;
+  BOOKABLE_ON_REQUEST = 2;
+}
```

Comment:
Added explanation for confirmation time. No need to add "status". This will be revisited when we add "option bookings" so not making it a boolean for now.